lib/Auth/Source/privacyidea.php

```diff
@@ -201,6 +210,7 @@ protected function login_chal_resp($username, $password, $transaction_id, $signa
 
         try {
             $result = $body->result;
+            $otpAttributes = $body->detail;
```

Comment:
again nitpicking: I think the variable name "otpAttributes" could be a bit misleading. the details section does not contain "OTP" information. It basically contains token information, but it could contain also any other stuff.